### DISCUSSION TITLE
Nextcloud and have a group maps to the Nextcloud admin group

### DISCUSSION THREAD

**Discussion by Smithoo4** [lldap/lldap#689] [Q&A] [answered]
I have gotten LLDAP to work with Nextcloud following the example: [https://github.com/lldap/lldap/blob/main/example_configs/nextcloud.md](url).

I was hoping to map an LDAP group (e.g. nextcloud_admins) to the admins group in Nextcloud. This way you can set the admins in LDAP. However, I don’t seem to be able to find away of doing this. Does anyone know how to do this? Or is it just not possible? I have a feeling it is a limitation on the Nextcloud side. I have asked but not heard anything back, [https://help.nextcloud.com/t/ldap-and-admins-group/169758](url)

Thank you

**Comment by nitnelave** [comment, marked as answer]
It looks like it's still blocked on nextcloud's side:
https://github.com/nextcloud/server/issues/6428

There's nothing I can do about it on LLDAP, and as soon as they implement
it, it should work with LLDAP.

I'll close this since I can do nothing but commiserate...

**Reply by Smithoo4** [reply]
This is too bad, but thank you for the response.
